content/ADR-183-favorite-places.md

```diff
@@ -0,0 +1,116 @@
+
+Following a list of the expected functionalities:
+
+- Users can add a place to their favorite from the explore menu
+- Users can navigate to a new tab listing all the places they added to favorites
+In a future iteration it will be possible to favorite places from the minimap HUD and also from the map parcel info pop-up, currently not possible due to technical limitations of the renderer map handling and Places API restriction to curated places.
+More in the limitations section.
+
+## Solution
+
+### Data storing
+
+After an initial discussion about whether to store favorites in the user profile or in Places, we are more incline to store everything in Places, more informations about the decision journey at the end.
+
+### Favorite retrieval
+
+The Places API will provide the required informations through:
+
+- An endpoint to obtain the (paginated if we decide for a big maximum amount) list of favorites.
+- An endpoint to add a new favorite for the current user (already available)
+- An endpoint to remove a favorite for the current user (already available)
+In additon a change will be required in the current places data structure in order to show the favorite information in the explore places list with a boolean. (already provided by the places API, but requires changes in browser-interface and unity-renderer)
+
+### Data structure
+
+To facilitate the addition of the feature when handling favorites we should use the same data structure currently used for Places, with the boolean addition previously mentioned:
+
+```
+public class Realm
+{
+	public string serverName;
+	public string layer;
+	public int usersCount;
+	public int maxUsers;
+	public Vector2Int[] userParcels;
+}
+
+public string id;
+public string name;
+public string creator;
+public string description;
+public string thumbnail;
+public Vector2Int baseCoords;
+public Vector2Int[] parcels;
+public int usersTotalCount;
+public bool isPOI;
+public Realm[] realms;
+public bool isFavorite;
+```
+
+## Current limitations
+
+With the current functionalities provided by the Places website it is not possible to favorite all the Decentraland areas due to a restriction that allows only curated Places to be displayed. This together with a legacy code integration of the map in the unity renderer will allow the favorite functionality to be applied only in the places section in the explore menu.
+
+## Decision Journey
+
+In the initial iteration it has been discussed where to store the favorite list, the two options were storing them in the user profile or storing them in Places, the following PROs and CONs made us lean towards the Places option:
+
+Storing favorite in the user profile:
+  - PROs
+    * Easily readable in unity renderer
+    * Simple to obtain favorites of other players
+    * Decentralised approach
+    * Favorites can be imported in places without user interaction
+  - CONs
+    * requires updates on the profile schema
```

Comment:
Just a comment here. I think adding a field in the profile schema is not required to make it work (it's an open schema allowing additional properties: https://github.com/decentraland/common-schemas/blob/main/src/platform/profile/profile.ts#L30)